### PR TITLE
Initial update script implementation

### PR DESCRIPTION
**Pull request by tobru**
Second try for a dead-simple update script, compatible with `/bin/sh` (no bash!).

### CODE REVIEW

File: update-packages.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+set -e
```
Comment (mhutter) [inline update-packages.sh]:
```suggestion
set -e -u -o pipefail
```
* `-u` - fail when using undefined variables
* `-o pipefail` - fail if one of the commands in a pipe fails

File: update-packages.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    exit 1
+  fi
+
+  log info "Package cache refresh finished"
+}
+
+apt_get_upgrade() {
+  log info "Doing package upgrade"
+  if ! apt-get -qy dist-upgrade --auto-remove --purge; then
+    log error "Package upgrade failed"
```
Comment (mhutter) [inline update-packages.sh:26]: Is there no `exit 1` on purpose?

**Review by mhutter — CHANGES_REQUESTED** [review]

File: update-packages.sh (hunk 1 above)
Comment (tobru) [reply update-packages.sh]: `-o pipefail` is not supported by `/bin/sh`. `-u` is added now.

File: update-packages.sh (hunk 2 above)
Comment (tobru) [reply update-packages.sh:26]: Missed that one. Thanks.

**Review by mhutter — APPROVED** [review]

**Review by simu — APPROVED** [review]
LGTM as a first iteration

**Pull request merged**